exercises/concept/lasagna-master/lasagna_master_test.cpp

```diff
@@ -11,26 +11,20 @@ using namespace lasagna_master;
 
 // As long as we have catch2 v2 and v3 in parallel, we can't use either
 //  of their comparison marcors for floating point comparisons.
-#define REQUIRE_VECTOR_APROX_EQUAL(vec1, vec2, margin)  \
-    REQUIRE(vec1.size() == vec2.size());                \
-    for (size_t i = 0; i < vec1.size(); i++) {          \
-        REQUIRE( std::abs(vec1.at(i) - vec2.at(i)) < margin);     \
-    } 
-
-TEST_CASE("preparationTime: Preparation time for many layers with custom average time", "[task_1]") {
+#define REQUIRE_VECTOR_APROX_EQUAL(vec1, vec2, margin)       \
+    REQUIRE(vec1.size() == vec2.size());                     \
+    for (size_t i = 0; i < vec1.size(); i++) {               \
+        REQUIRE(std::abs(vec1.at(i) - vec2.at(i)) < margin); \
+    }
+
+TEST_CASE(
+    "preparationTime: Preparation time for many layers with custom average "
+    "time",
```

Comment:
Until https://github.com/exercism/cpp-test-runner/issues/73 is fixed, we will need some more `// clang-format off` for cases like this.

Reply:
Updated this now.